contributors/design-proposals/propagation.md

```diff
@@ -73,13 +79,18 @@ type VolumeMount struct {
 	// Required.
 	MountPath string `json:"mountPath"`
 	// Optional.
-	Propagation string `json:"propagation"`
+	Propagation PropagationMode `json:"propagation,omitempty"`
```

Comment:
This field name is very vague. So is the file name, by the way. Propagation of what? In what direction?